tests/stdlib/tasynchttpserver_transferencoding.nim

```diff
@@ -1,6 +1,9 @@
 import httpclient, asynchttpserver, asyncdispatch, asyncfutures
 import net
 
+import std/asyncnet
+import std/nativesockets
+
 const postBegin = """
 POST / HTTP/1.1
 Host: 127.0.0.1:64123
```

Comment:
test passes even though we're now using dynamic port, is that expected to pass?

Reply:
Interesting observation. It is unrelated to my change, so I'm going to say that yes, it is expected to pass. It seems that the address is passed down the chain and originates from https://github.com/nim-lang/Nim/blob/fd5c8ef20845a511ad4e0af8dd8ad4331bf46ffc/lib/pure/asynchttpserver.nim#L382

Nothing else verifies or validates it currently. 

This may be a desirable feature (ex. if the Nim server is running behind a proxy and the proxy doesn't rewrite the header but keeps the external IP).

Reply:
I removed everything but the `POST` line and `Transfer-Encoding: chunked` and it still worked, so it seems reasonable to remove it all so as to not confuse anyone in the future.